Sources/PointFree/Auth.swift

```diff
@@ -359,7 +359,9 @@ private func gitHubAuthTokenMiddleware(
   do {
     let gitHubUser = try await gitHub.fetchUser(accessToken)
     let user = try await fetchOrRegisterUser(accessToken: accessToken, gitHubUser: gitHubUser)
-    try await refreshStripeSubscription(for: user)
+    await notifyError("GitHub Auth: Refresh stripe failed") {
+      try await refreshStripeSubscription(for: user)
+    }
```

Comment:
We will still notify us when there is a stripe problem on login, we just won't fail login.